terraform.tfstate.backup

```diff
@@ -0,0 +1,398 @@
+{
+  "version": 4,
+  "terraform_version": "1.0.9",
+  "serial": 10,
+  "lineage": "e87e0ecb-c1ec-3f60-72d7-497b4aaee798",
+  "outputs": {
+    "cd_webhook": {
+      "value": "https://$JB-ToDo-App-Mod12:avdrg5BtFa1q55rNE6jRekcdS9EPbkAgM48fplXJkL8liuPx9Q9Pu3AeJzEM@JB-ToDo-App-Mod12.scm.azurewebsites.net/docker/hook",
```

Comment:
Looks like this snuck through before you updated your `.gitignore` - and so did some of the sensitive values stored in it! We should remove this (`git rm --cached terraform.tfstate.backup`) and make sure we cycle any secrets mentioned in this doc

Reply:
removed  tfstate.backup - git rm --cached terraform.tfstate.backup